### DISCUSSION TITLE
Ability to change the Nemmera APC wreck removal vehicle for another class

### DISCUSSION THREAD

**Discussion by TheLocalPub** [Vdauphin/HeartsAndMinds#1605] [Question and answer] [answered]
I've looked through, but can't seem to find anything referring to the Nemmera in the mission apart from in removerubbish.sqf, but this is nothing more then a comment.

Is there a possibility to change which vehicle can clear wrecks, ieds, etc?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

You can tweak it here: https://github.com/Vdauphin/HeartsAndMinds/blob/1.24.0/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/ied/deleteLoop.sqf#L31

Cheers

**Reply by TheLocalPub** [reply]
Perfect! Just what I wanted :)
